### ISSUE TITLE
small documentation updates

### ISSUE THREAD

**Issue by mmitch** [mmitch/cobol-travis#11]
`BUILD BINARY`:
- check backwards pointing arrow layout - one more dash?
- split _source file_ and _main source file_?

**Comment by mmitch** [comment]
arrow layout checked and fixed with 58a890e

**Comment by mmitch** [comment]
documentation is already correct.  I use this:

```
  <---------|
--- source -----
```

That is already "one or more", there is no way for "none".
"none, one or more" would be this:

```
  <---------|
--- source -----
 |----------|
 ```
